README.md

```diff
@@ -6,42 +6,47 @@ Reactive libraries for Angular
 
-
 ## Packages
 
-- [@ngrx/store](./docs/store/README.md) - RxJS powered state management for Angular applications, inspired by Redux  
-- [@ngrx/effects](./docs/effects/README.md) - Side Effect model for @ngrx/store to model event sources as actions.   
-- [@ngrx/router-store](./docs/router-store/README.md) - Bindings to connect the Angular Router to @ngrx/store  
-- [@ngrx/store-devtools](./docs/store-devtools/README.md) - Store instrumentation that enables a
-[powerful time-travelling debugger](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en).
-- [@ngrx/entity](./docs/entity/README.md) - Entity State adapter for managing record collections.
-- [@ngrx/schematics](./docs/schematics/README.md) - Scaffolding library for Angular applications using NgRx.
+* [@ngrx/store](./docs/store/README.md) - RxJS powered state management for Angular applications, inspired by Redux
+* [@ngrx/effects](./docs/effects/README.md) - Side Effect model for @ngrx/store to model event sources as actions.
+* [@ngrx/router-store](./docs/router-store/README.md) - Bindings to connect the Angular Router to @ngrx/store
+* [@ngrx/store-devtools](./docs/store-devtools/README.md) - Store instrumentation that enables a
+  [powerful time-travelling debugger](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en).
+* [@ngrx/entity](./docs/entity/README.md) - Entity State adapter for managing record collections.
+* [@ngrx/schematics](./docs/schematics/README.md) - Scaffolding library for Angular applications using NgRx.
 
 ## Examples
-- [example-app](./example-app/README.md) - Example application utilizing @ngrx libraries, showcasing common patterns and best practices.  Try it on [StackBlitz](https://stackblitz.com/github/ngrx/platform/).
+
+* [example-app](./example-app/README.md) - Example application utilizing @ngrx libraries, showcasing common patterns and best practices. Try it on [StackBlitz](https://stackblitz.com/github/ngrx/platform/).
 
 ## Migration
-- [Migration guide](./MIGRATION.md) for users of ngrx packages prior to 4.x.
+
+* [Migration guide](./MIGRATION.md) for users of ngrx packages prior to 4.x.
 
 ## Events
-Join us at ngATL in Atlanta, Georgia on January 30 - February 2. [Read here to learn more](http://ng-atl.org/) or [use this special offer to buy one ticket and get another free](https://t.co/2071F6VvbV)
+
+* [ng-conf](https://ng-conf.org/) in Salt Lake City, Utah on April 18 - April 20th.
+* [We RISE Tech](https://werise.tech/) in Atlanta, Georgia on June 21 - 22nd.
+* [Framework Summit](https://frameworksummit.com/) in Park City, Utah on October 2 - 3rd.
+* [AngularMix](https://www.angularmix.com/) in Orlando, Florida on October 10 - 12th. Use the discount code "ryan" or "roberts" to receive $50 off your registration!
```

Comment:
Will you add Angular Connect after this one?

Reply:
Done